src/pages/players.js

```diff
@@ -0,0 +1,157 @@
+var emojify = require('../lib/emojify')
+var shortid = require('shortid')
+
+function list(templates, season, player, req, res) {
+  var season_id = emojify.unemojify(req.params.season_id)
+  season.getSeason(season_id).then(season => {
+    season.vanity = emojify.emojify(season.id)
+    return player.getPlayers(season_id).then(players => {
+      players = players.map(player => {
+        player.vanity = emojify.emojify(player.id)
+        player.season_vanity = emojify.emojify(player.season_id)
+        return player
+      })
+      var html = templates.player.list({
+        user: req.user,
+        season: season,
+        players: players
+      })
+
+      res.send(html)
+    })
+  }).catch(err => {
+    console.error(err)
+    res.sendStatus(500)
+  })
+}
+
+function create(templates, season, steam_user, req, res) {
+  if (!req.user || !req.user.isAdmin) {
+    res.sendStatus(403)
+    return
+  }
+
+  var season_id = emojify.unemojify(req.params.season_id)
+
+  season.getSeason(season_id).then(season => {
+    return steam_user.getSteamUsers().then(steamUsers => {
+      var html = templates.player.edit({
+        user: req.user,
+        verb: 'Create',
+        season: season,
+        steamUsers: steamUsers
+      })
+
+      res.send(html)
+    })
+  }).catch(err => {
+    console.error(err)
+    res.sendStatus(500)
+  })
+}
+
+function edit(templates, season, player, steam_user, req, res) {
+  if (!req.user || !req.user.isAdmin) {
+    res.sendStatus(403)
+    return
+  }
+
+  var season_id = emojify.unemojify(req.params.season_id)
+  var id = emojify.unemojify(req.params.id)
+
+  season.getSeason(season_id).then(season => {
+    return steam_user.getSteamUsers().then(steamUsers => {
+      return player.getPlayer(id).then(player => {
+        player.vanity = emojify.emojify(player.id)
+        var html = templates.player.edit({
+          user: req.user,
+          verb: 'Edit',
+          player: player,
+          season: season,
+          steamUsers: steamUsers
+        })
+        res.send(html)
+      })
+    })
+  }).catch(err => {
+    console.error(err)
+    res.sendStatus(500)
+  })
+}
+
+function post(player, req, res) {
+  if (!req.user || !req.user.isAdmin) {
+    res.sendStatus(403)
+    return
+  }
+
+  var season_id = req.body.season_id
+  var season_vanity = emojify.emojify(season_id)
+  var id = req.body.id ? req.body.id : shortid.generate()
+  var p = req.body
+  p.id = id
+  if (p.will_captain === 'on') {
```

Comment:
```
p.will_captain = p.will_captain === 'on' 
```
is 1 line rather than 4
Also, what about when p,will_captain has already been updated once?  true !== 'on' so that'd be turned into false so what you probably want is actually:

```
p.will_captain = p.will_captain === 'on' || p.will_captain
```

Put whichever you think will be true most often first.